JoinColony/colonyDapp
# Add synaps integration

## Description

This PR adds in synaps verification for metacolony

**New stuff** ✨

* Synaps kyc dialog
* Kyc oracle api

**Changes** 🏗

* Updated Getwhitelisted component to show synaps dialog when metacolony
* Refactored auth/api to be used also for kyc endpoint

**How to test** ⚰️
NOTE 
- you need to have access to synaps.io (ask jack)
- synaps integration is only for metacolony so testing locally will be hacky

* Go to colonyNetwork/packages/kyc-oracle and checkout to kyc-oracle-update
* Run: ```node ./bin/index.js --adminAddress 0x123...456 --whitelistAddress 0xabc...def --apiKey IamASynapsAPIKey --dbPath ./kycSssions``` with proper var
* Install/enable whitelist with kyc, install/enable coin machine
* Force opening synaps modal in GetWhitelisted component
* You should see synaps iframe
* Fill all needed data and "validate" this user from synaps.io
* You should see iframe replaced with content "You are verified"

Resolves DEV-471
